package.json

```diff
@@ -22,7 +22,7 @@
     "build-index": "npm run build-package && node tasks/generate-index",
     "build-legacy": "shx rm -rf build && npm run build-index && webpack --config config/webpack-config-legacy-build.js && cleancss --source-map src/ol/ol.css -o build/legacy/ol.css",
     "copy-css": "shx cp src/ol/ol.css build/ol/ol.css",
-    "transpile": "shx rm -rf build/ol && shx mkdir -p build/ol && shx cp -rf src/ol build/ol/src && node tasks/serialize-workers && tsc --project config/tsconfig-build.json",
+    "transpile": "shx rm -rf build/ol && shx mkdir -p build/ol && shx cp -rf src/ol build/ol/src && node tasks/serialize-workers && npm install --no-save typescript@3.8.1-rc && tsc --project config/tsconfig-build.json && npm install",
```

Comment:
I think this could be `npx --package typescript @3.8.1-rc tsc --project config/tsconfig-build.json` (`npx` comes with `npm@5.2+`.  Not sure that's really better or worse though.

Reply:
I like it. Force-pushed.